docs/topics/api/addons.rst

```diff
@@ -957,7 +957,7 @@ Replacement Add-ons
 
 .. _addon-replacement-addons:
 
-This endpoint returns a list of suggested replacements for legacy add-ons that are unsupported in Firefox 57.  Added to support the TAAR recommendation service.
```

Comment:
Could you add this to the addons DRF viewset in the relevant method? Just good hygiene to continually update the methods so we eventually don't have to maintain the separate api docs.

Reply:
Can you link to docs explaining how to do this?